src/vm/wren_core.c

```diff
@@ -666,6 +667,21 @@ DEF_PRIMITIVE(num_sign)
   }
 }
 
+DEF_PRIMITIVE(num_highest)
+{
+  RETURN_NUM(DBL_MAX);
+}
+
+DEF_PRIMITIVE(num_lowest)
+{
+  RETURN_NUM(-DBL_MAX);
```

Comment:
Why not DBL_MIN? they have different values in the header.

> DBL_MIN 2.22507385850720138309e-308L
> DBL_MAX 1.79769313486231570815e+308L

(or are they identical, and it's just floating point _error_?)